GetFed/GetFed/Controllers/FoodDetailViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  FoodDetailViewController.swift
+//  GetFed
+//
+//  Created by Britney Smith on 11/26/18.
+//  Copyright © 2018 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+class FoodDetailViewController: UIViewController {
+
+    // MARK - Outlets
+    
+    @IBOutlet var foodNameLabel: UILabel!
+    @IBOutlet var caloriesLabel: UILabel!
+    @IBOutlet var brandNameLabel: UILabel!
+    
+    // MARK - Properties
+    var food: Food?
+
+    
+    // MARK - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        populateLabels()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.isNavigationBarHidden = false
+    }
+    
+    // MARK - Methods
+    
+    func populateLabels() {
+        guard let food = food,
```

Comment:
So if you don't have all of these bits of data, the screen is just blank? It's probably a good idea to consider how the app should look when certain fields are missing.

Reply:
Agreed! I've added more robust handling in my current pull request (commit 
23938e8).